Application/EdFi.Ods.SandboxAdmin/EdFi.Ods.SandboxAdmin.csproj

```diff
@@ -31,6 +31,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="Npgsql" Version="6.0.3" />
     <PackageReference Include="NHibernate" Version="5.3.11" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.6" />
```

Comment:
Shouldn't this be a hangfire upgrade rather than addition of System.Data.SqlClient ?

Reply:
HangFire Updated